src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+// import bootstrap from "bootstrap";
```

Comment:
- [ OPTIONAL] Please ensure that any unused commented codes are removed from the file. Thank you.